src/react-intl-messages.ts

```diff
@@ -533,6 +535,8 @@ export default {
         [FM.ACTIONCREATOREDITOR_WARNING_PAYLOAD]: 'Bot Response is Required',
         [FM.ACTIONCREATOREDITOR_WARNING_ENTITY]: 'Removed reference to undefined Entity',
         [FM.ACTIONCREATOREDITOR_WARNING_NONEMPTYFIELD]: `Non-wait Actions can't have Expected Entities`,
+        [FM.ACTIONCREATOREDITOR_WARNING_DUPLICATEACTION]: `Actions must be unique`,
```

Comment:
Hmm, I think some of our error messages are worded this way and I prefer this "[noun] must be [condition]" formate, but since it's shown to the user I think we should try to make it be more actionable and friendly.
I would prefer something like this:

"The action you attempted to create has the same values as another action that already exists on your model. All actions must be unique. Please change the properties or close."

Can probably word it better but point is it tell them what they can do.  Either change the a property or close and discard the action.

Reply:
Reworded for a more positive tone

Reply:
I see you changed `FM.ACTIONCREATOREDITOR_WARNING_DUPLICATEFOUND`, but left `FM.ACTIONCREATOREDITOR_WARNING_DUPLICATEACTION`.  Why?

It looks like the `FM.ACTIONCREATOREDITOR_WARNING_DUPLICATEACTION` is showing to the user in the Confirmation modal up top: https://github.com/Microsoft/ConversationLearner-UI/pull/1012/files#diff-d106c181edc321d094ccd9d2a2c34667R1421